server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/Application.java

```diff
@@ -14,7 +14,10 @@
  */
 public class Application {
 
+    //private static int counter;
```

Comment:
Why is this here? Same with the other counter-related additions

Reply:
already deleted